aws-secretsmanager/src/test/groovy/io/micronaut/aws/secretsmanager/SecretsManagerConfigurationSpec.groovy

```diff
@@ -5,7 +5,7 @@ class SecretsManagerConfigurationSpec  extends ApplicationContextSpecification {
     @Override
     Map<String, Object> getConfiguration() {
         super.configuration + [
-                'aws.secretsmanager.enabled': false
+                'aws.services.secretsmanager.enabled': false
```

Comment:
This is a breaking change. I understand the reason why you changed it (agree that consistency is good), but it being a breaking change would require a new major release of micronaut-aws.

I suggest keeping them as they are for now.